File: Blueprints/BlueprintDefinitions/netcore3.1/CustomRuntimeFunction-FSharp/template/test/BlueprintBaseName.1.Tests/FunctionTest.fs
namespace BlueprintBaseName._1.Tests


open Xunit
open Amazon.Lambda.TestUtilities
open BlueprintBaseName._1


module FunctionTest =

    [<Fact>]
    let ``Invoke ToUpper Lambda Function``() =
        // Invoke the lambda function and confirm the string was upper cased.
        let context = TestLambdaContext()
        let upperCase = Function.functionHandler "hello world" context

        Assert.Equal("HELLO WORLD", upperCase)

    [<EntryPoint>]
    let main _ = 0

File: Blueprints/BlueprintDefinitions/vs2026/EmptyFunction-Image/template/test/BlueprintBaseName.1.Tests/FunctionTest.cs
using Xunit;
using Amazon.Lambda.Core;
using Amazon.Lambda.TestUtilities;

namespace BlueprintBaseName._1.Tests;

public class FunctionTest
{
    [Fact]
    public void TestToUpperFunction()
    {

        // Invoke the lambda function and confirm the string was upper cased.
        var function = new Function();
        var context = new TestLambdaContext();
        var casing = function.FunctionHandler("hello world", context);

        Assert.Equal("hello world", casing.Lower);
        Assert.Equal("HELLO WORLD", casing.Upper);
    }
}